app/src/main/dist/config/crate.yml

```diff
@@ -478,18 +477,11 @@
 #discovery.zen.publish_timeout: 30s
 
 # Unicast discovery allows to explicitly control which nodes will be used
-# to discover the cluster. It can be used when multicast is not present,
-# or to restrict the cluster communication-wise.
-# For example, Amazon Web Services doesn't support multicast discovery.
-# Therefore, you need to specify the instances you want to connect to a
-# cluster as described in the following steps:
+# to discover the cluster. Therefore, you need to specify the instances you want
+# to connect to a cluster as described in the following steps:
```

Comment:
are these really still steps? Isn't this just one? if so, maybe rephrase the sentence..